app/channels.go

```diff
@@ -77,6 +77,9 @@ type Channels struct {
 
 	postReminderMut  sync.Mutex
 	postReminderTask *model.ScheduledTask
+
```

Comment:
I did this here because plugins data was already here. Alternative is [here](https://github.com/mattermost/mattermost-server/blob/07f04b0349f21e122190b9596c8714b64daef96c/app/app.go#L27)

Reply:
Indeed, perhaps it's the only viable place? Not sure if we should sync with server platform to put this in the "right place" or if the multi-product architecture doesn't yet have a solution for this.